.github/workflows/cuda.yml

```diff
@@ -27,14 +27,32 @@ jobs:
             compiler: gcc
             python_version: "3.8"
             cuda_version: "11.5.1"
+            tree_learner: cuda
           - method: pip
             compiler: clang
             python_version: "3.9"
             cuda_version: "10.0"
+            tree_learner: cuda
           - method: wheel
             compiler: gcc
             python_version: "3.10"
             cuda_version: "9.0"
+            tree_learner: cuda
+          - method: source
+            compiler: clang
+            python_version: "3.8"
+            cuda_version: "11.5.1"
+            tree_learner: cuda_exp
+          - method: pip
+            compiler: clang
+            python_version: "3.9"
+            cuda_version: "10.0"
+            tree_learner: cuda_exp
+          - method: wheel
+            compiler: gcc
+            python_version: "3.10"
+            cuda_version: "11.5.1"
+            tree_learner: cuda_exp
```

Comment:
All our CUDA CI jobs are run on our own server sequentially. There is no any parallelization there. There are 3 CUDA (old) jobs because each of them tests different major CUDA version. WDYT about removing this one CUDA_EXP CI job  with the aim to decrease overall CI time?
```suggestion
```

Reply:
Sure. We can leave only two CI jobs for CUDA_EXP.